README.md

```diff
@@ -66,6 +66,36 @@ $ make
 On successful compilation, the “libtrfgen.so” library gets generated under
 Trfgen/lib folder.
 
+# Testing with Magma
+Following points should be considered when using S1APTester with
+[Magma](https://github.com/facebookincubator/magma)
+
+## UE IP Address Configuration
+While testing with [Magma](https://github.com/facebookincubator/magma) setup,
+the current configuration parameters in Magma allow allocation of only 243 UE
+IP addresses. We need to change the configuration in Magma codebase, in order
+to support allocation of more than 243 UE IP addresses.
+
+The following mask value, i.e., 24 needs to be changed in the file:
+[s1ap_wrapper.py](https://github.com/facebookincubator/magma/blob/master/lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py)
+(Filepath: [magma/lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py](https://github.com/facebookincubator/magma/blob/master/lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py))
+```
+TEST_IP_BLOCK = "192.168.128.0/24"
+```
+
+Magma has reserved 12 IP addresses for internal testing purpose and therefore
```

Comment:
11 IP addresses reserved for Magma internal. Other 2 are all zeros and all ones that are not allocatable.

Reply:
Updated the README file with this information.